src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -101,18 +105,18 @@ public class S3BlobStoreConfig extends AbstractAwsGlobalConfiguration {
     private final boolean deleteArtifacts;
     
     private final boolean deleteStashes;
-    
+
     /**
      * class to test configuration against Amazon S3 Bucket.
      */
     private static class S3BlobStoreTester extends S3BlobStore {
         private static final long serialVersionUID = -3645770416235883487L;
+
+        @SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "This transient field is only modified from the class constructor.")
         private transient S3BlobStoreConfig config;
 
-        S3BlobStoreTester(String container, String prefix, boolean useHttp,
-            boolean useTransferAcceleration, boolean usePathStyleUrl,
-            boolean disableSessionToken, String customEndpoint,
-            String customSigningRegion) {
+        S3BlobStoreTester(String container, String prefix, boolean useHttp, boolean useTransferAcceleration, boolean usePathStyleUrl,
+                          boolean disableSessionToken, String customEndpoint, String customSigningRegion) {
```

Comment:
Please avoid gratuitous formatting changes.

Reply:
Reverted, sorry for that.